catalog/app/components/Markdown/Markdown.js

```diff
@@ -12,10 +13,13 @@ import { withStyles } from '@material-ui/core/styles';
 import { linkStyle } from 'utils/StyledLink';
 import * as RT from 'utils/reactTools';
 
+const SANITIZE_OPTS = {
+  'FORBID_TAGS': ['style', 'script'],
```

Comment:
I think we should really be using a whitelist. Theoretically with a blacklist we can miss dangerous tags in sanitization. There's also the theoretical vulnerability of new tags being added to the HTML standard that we're not aware of.

In general, I'm surprised how manual this process is. I was pretty sure that there are HTML sanitization libraries that do the right thing in one function, without requiring you to specify the tags you don't want to allow yourself.

Reply:
i second this, we should use a whitelist (a short one)

Reply:
@ResidentMario judging by the docs, this lib is pretty awesome and it does the necessary xss protection by default, so these forbidden tags and attrs are extra